app/assets/javascripts/bundle-import-helpers.js

```diff
@@ -1,195 +0,0 @@
-environments within a graphical interface (https://launchpad.net/juju-gui).
-Copyright (C) 2012-2013 Canonical Ltd.
-
-This program is free software: you can redistribute it and/or modify it under
-the terms of the GNU Affero General Public License version 3, as published by
-the Free Software Foundation.
-
-This program is distributed in the hope that it will be useful, but WITHOUT
-ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
-SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
-General Public License for more details.
-
-You should have received a copy of the GNU Affero General Public License along
-with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
-
-'use strict';
-
-YUI.add('bundle-import-helpers', function(Y) {
-  var ns = Y.namespace('juju');
-
-  // Maps bundle deployment statuses to notification messages.
-  var STATUS_MESSAGES = {
-    scheduled: 'The deployment has been scheduled and is now pending',
-    started: 'The deployment is currently in progress',
-    completed: 'The deployment has been successfully completed',
-    cancelled: 'The deployment has been cancelled'
-  };
-
-  ns.BundleHelpers = {
-    /**
-      Start watching scheduled/started bundle deployments.
-      Notify recently occurred deployment errors.
-
-      @method watchAll
-      @param {Environment} env The environment object exposing the Juju API.
-      @param {Database} db The db which contains the NotificationList used
-        for adding notifications to the system.
-    */
-    watchAll: function(env, db) {
-      env.deployerStatus(function(response) {
-        if (response.err) {
-          db.notifications.add({
-            title: 'Unable to retrieve bundle deployment statuses',
-            message: 'Failure retrieving bundles status: ' + response.err,
-            level: 'error'
-          });
-          return;
-        }
-        var timestamp = (Date.now() / 1000) - (60 * 60); // One hour ago.
-        response.changes.forEach(function(change) {
-          var status = change.status;
-          // Start watching scheduled/in progress bundle deployments.
-          if (status === 'scheduled' || status === 'started') {
-            ns.BundleHelpers._watchDeployment(change.deploymentId, env, db);
-            return;
-          }
-          // Notify bundle deployment errors occurred in the last hour.
-          if (change.err && change.time >= timestamp) {
-            ns.BundleHelpers._notifyDeploymentChange(
-                db, change.deploymentId, status, change.err);
-          }
-          // If none of the previous blocks detect any changes then ignore
-          // successfully completed/old/canceled deployments.
-        });
-      });
-    },
-
-    /**
-    Export the YAML for the environment.
-
-    @method exportYAML
-    @param {Database} db The app Database with access to the
-      NotificationList.
-    */
-    exportYAML: function(db) {
```

Comment:
Okay sounds like the other branch needs  to land before this one can be
thoroughly Qa'd
On May 11, 2015 1:30 PM, "Jeff Pihach" <notifications@github.com> wrote:

> In app/assets/javascripts/bundle-import-helpers.js
> <https://github.com/juju/juju-gui/pull/730#discussion_r30061474>:
>
> > -                db, change.deploymentId, status, change.err);
> > -          }
> > -          // If none of the previous blocks detect any changes then ignore
> > -          // successfully completed/old/canceled deployments.
> > -        });
> > -      });
> > -    },
> > -
> > -    /**
> > -    Export the YAML for the environment.
> > -
> > -    @method exportYAML
> > -    @param {Database} db The app Database with access to the
> > -      NotificationList.
> > -    */
> > -    exportYAML: function(db) {
>
> This functionality was still needed so it was moved to the deployer bar
> where it was primarily used from.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/juju/juju-gui/pull/730/files#r30061474>.
>